### PR TITLE
MINOR: Add Lucas Bradstreet to committers

### PR DESCRIPTION
**Pull request by lbradstreet**
I recently became a committer https://projects.apache.org/committee.html?kafka.

### CODE REVIEW

File: committers.html [hunk 1]
```diff
@@ -552,6 +552,13 @@ <h1 class="content-title">The committers</h1>
           <a href="https://github.com/gharris1727/">github.com/gharris1727</a><br>
           <a href="https://twitter.com/gharris1727">@gharris1727</a><br>
         </td>
+        <td>
+          Lucas Bradstreet<br>
```
Comment (jolshan) [inline committers.html:559]: Did you leave out your picture here?

Comment (lbradstreet) [reply committers.html:559]: Oops, should have opened the html.

**Review by jolshan — APPROVED** [review]

**Comment by lbradstreet** [comment]
@jolshan if you or someone else is able to merge this it’d be helpful. This PR is what should get me the ability to merge :)

**Comment by jolshan** [comment]
Hmm - you should be able to merge this too. If you don't you may want to check your permissions. 😅  (See I merged my own -- https://github.com/apache/kafka-site/pull/473)

But I can go ahead and merge.

**Pull request merged**